sdk/src/main/java/io/dapr/client/AbstractDaprClient.java

```diff
@@ -392,6 +396,34 @@ public <T> Mono<QueryStateResponse<T>> queryState(QueryStateRequest request, Cla
     return this.queryState(request, TypeRef.get(clazz));
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public <T> Mono<BulkPublishResponse<T>> publishEvents(String pubsubName, String topicName, List<T> events,
```

Comment:
nit: instead, can you move content type prior to events? It allows a overloaded method using `T... events` as an array.

Reply:
done, changed the other func with reqMeta also.